src/main.js

```diff
@@ -145,6 +150,37 @@ class PercyClient {
     return this._httpPost(`${this.apiUrl}/builds/${buildId}/resources/`, data);
   }
 
+  uploadResources(buildId, resources) {
+    const _this = this;
+    function* generatePromises() {
+      for (const resource of resources) {
+        const content = resource.localPath
+          ? fs.readFileSync(resource.localPath, 'utf8')
```

Comment:
This is not guaranteed to be 'utf8' encoded, it might be an image.

Reply:
I think you can just use `fs.readFileSync(resource.localPath)`

Reply:
If you don't specify the encoding you get a buffer back rather than a string, can the uploadResource function handle a buffer?

Reply:
Yes!

Reply:
uploadResource always runs content this method: https://github.com/percy/percy-js/blob/bcfc11e9e4d235f2283d33b1a275c062fbf395bb/src/utils.js#L11